cmd/tailscale/cli/file.go

```diff
@@ -156,6 +159,12 @@ func runCp(ctx context.Context, args []string) error {
 		if cpArgs.verbose {
 			log.Printf("sending %q to %v/%v/%v ...", name, target, ip, stableID)
 		}
+		if cpArgs.progress {
+			rc := progress.New(fileContents, contentLength, time.Second)
+			defer rc.Close()
```

Comment:
This is happening in a loop, processing one file after another. `defer` in a loop smells a bit weird. If you use this to `cp` multiple files, does the progress bar do the right thing?

Reply:
Good call; yes, this does misbehave. Just pushed a commit fixing this and tested locally with two files.